### PR TITLE
[FIX] bill_configuration.rst: remove paragraph talking about routings…

### PR DESCRIPTION
**Pull request by Danisan**
[FIX] bill_configuration.rst: remove paragraph talking about routings. Replace it with operations.
Add timer and add locking quantities section.
Add images with configuration options.

### CODE REVIEW

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
@Lau-Odoo can you do a review please?
Also, is this only for v14?

a few points about the images already:
- the resolution of the images is way too high. You could shrink to 50%
- [compress them with pngquant](https://www.odoo.com/documentation/14.0/contributing/documentation.html#pngquant)
- [use alt tags](https://www.odoo.com/documentation/14.0/contributing/documentation/content_guidelines.html#alt-tags)
- put them in the right [media folder] (https://www.odoo.com/documentation/14.0/contributing/documentation/content_guidelines.html#media-files) and name the files with words rather than numbers. (you could also move the old media files in this doc to the new media folder
- [align the indentation](https://www.odoo.com/documentation/14.0/contributing/documentation/rst_guidelines.html#be-consistent-with-indentation) of for the images options

Also:
- [break the lien at 100th character](https://www.odoo.com/documentation/14.0/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character)

File: content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst [hunk 1]
```diff
@@ -51,28 +51,50 @@ component.
...
-select when, in the work operations, your components are consumed with
-the dropdown menu.
+Now that you have created some work centers, add operations to your *BoM*,
+defining a work center for each of them. You can select when, in the work
+operations, your components are consumed with the dropdown menu.
 
 .. image:: media/bills_of_materials_04.png
     :align: center
 
+Adding the timer to work orders
```
Comment (weka-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst:72]: Title is misleading - the timer is already a feature of the work orders, not something you can add on?

Maybe change to "Timing your work orders"

Comment (weka-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst:72]: Adding a timer to work orders

File: content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst [hunk 2]
```diff
@@ -51,28 +51,50 @@ component.
...
+===============================
+
+You can add a timer to your work orders. This consists of tree buttons that
```
Comment (weka-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst:75]: "You can time your work orders using buttons that allow you to start, pause, or block each operation. Using these buttons will register the duration of each operation in real time."

Comment (weka-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst:75]: "The timer consists of three buttons that allow you to start, pause, or block each operation and registers the real duration of each operation.

File: content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst [hunk 3]
```diff
@@ -51,28 +51,50 @@ component.
...
+will allow you tu start, pause, block or get each operation done, thus registering
+the real duration of each operation.
+
+.. image:: media/bills_of_materials_07.png
+    :align: center
+
+You will also see the timer, in the manufacturing orders, when clicking the
```
Comment (weka-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst:82]: not sure if this sentence is necessary

Comment (weka-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst:82]: not sure if this sentence is necessary

File: content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst [hunk 4]
```diff
@@ -51,28 +51,50 @@ component.
...
+Work Orders tab.
+
+.. image:: media/bills_of_materials_08.png
+    :align: center
+
+Locking quantities consumed
+===========================
+
+Enabling this option, you can define that manufacturing orders are locked by
```
Comment (weka-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst:91]: Make sure to mention how/where to enable this option. 

i.e. "You can lock quantities to consume under Configuration > Settings.  Enabling this option will prevent manufacturing users from modifying quantities of components to consume."

Comment (weka-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst:91]: Make sure to mention how/where to enable this option. 

i.e. "You can lock quantities to consume under Configuration > Settings.  Enabling this option will prevent manufacturing users from modifying quantities of components to consume."

**Review by weka-odoo — CHANGES_REQUESTED** [review]

**Comment by jcs-odoo** [comment]
Hi,

I'm closing this old pull request as I see this file has been updated several times since then.

cc @Felicious @brse-odoo @hojo-odoo If this may be of interest for you, feel free to reopen this PR.

Thanks and have a good day!

**Pull request closed without merging**